### DISCUSSION TITLE
Is it possible to use client-id with plain request?

### DISCUSSION THREAD

**Discussion by trmdi** [AdguardTeam/AdGuardHome#5867] [General]
I managed to setup DoT through Traefik reverse proxy, but queries appear to be plain requests to Adguard. How can I send the client-id to Adguard when I use `tls://client-id.dns.example.com` ?

**Comment by trmdi** [comment]
Maybe relates to #2798 ?

**Comment by ainar-g** [comment]
Not currently, but it could be supported through an EDNS(0) extension.  Please file a feature request if you would like that.
